server/datastore/mysql/testing_utils.go

```diff
@@ -67,6 +67,14 @@ func connectMySQL(t testing.TB, testName string, opts *DatastoreTestOptions) *Da
 	tc := config.TestConfig()
 	tc.Osquery.MinSoftwareLastOpenedAtDiff = defaultMinLastOpenedAtDiff
 
+	// TODO: for some reason we never log datastore messages when running integration tests, why?
+	//
+	// Changes below assume that we want to follows the same pattern as the rest of the codebase.
+	dslogger := log.NewLogfmtLogger(os.Stdout)
```

Comment:
:+1: Makes sense to me. Not sure we log super often at the Datastore layer but I don't see how that would hurt.